src/bin/jcli_app/rest/v0/node/stats.rs

```diff
@@ -10,13 +10,19 @@ pub enum Stats {
         addr: HostAddr,
         #[structopt(flatten)]
         debug: DebugFlag,
+        #[structopt(flatten)]
+        output_format: OutputFormat,
     },
 }
 
 impl Stats {
     pub fn exec(self) {
-        let (addr, debug) = match self {
-            Stats::Get { addr, debug } => (addr, debug),
+        let (addr, debug, output_format) = match self {
+            Stats::Get {
+                addr,
+                debug,
+                output_format,
+            } => (addr, debug, output_format),
         };
```

Comment:
Could do it with less code:

```rest
let Stats::Get { addr, debug, output_format } = self;
```

Reply:
That's brilliant!

Reply:
That's destructuring `let` :smile:

Reply:
But it's done on a every possible (only) variant of an enum at once, I didn't know that compiler was able to do that

Reply:
Yeah, it only works because there is a single variant, so the match is irrefutable.
You can also use the same syntax with structs, similar to how the cool kids do it in JavaScript these days.